src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -543,6 +544,9 @@ verifyMethodSpec cc methodSpec lemmas checkSat tactic asp =
      (args, assumes, env, globals2) <-
        io $ verifyPrestate opts cc methodSpec globals1
 
+     when (detectVacuity opts)
```

Comment:
does this check cover `unsafe_assume`?

Reply:
I'm not sure I understand what you're referring to.

Do you mean that you would want for calls to `crucible_llvm_unsafe_assume_spec` to result in a check of that specification for vacuity?

Reply:
I think that's how I'd interpret it.

Reply:
I looked into doing this, but it brings up an awkward change for the API: in order to know what solver backend to use for checking the `crucible_llvm_unsafe_assume_spec`, we'd need to pass in a solver to that command, like it's currently done with `crucible_llvm_verify`.

Unless this argument was made optional, this would look weird as the command is not really about calling a solver.

Reply:
yes, it's unfortunate that `crucible_llvm_unsafe_assume_spec` can't do the same check, and adding an optional parameter is not really possible in saw-script. Let's make an issue to do that in python, when python is ready to use.

Reply:
#974